core/src/commonMain/kotlin/com/sourcepoint/mobile_core/models/consents/State.kt

```diff
@@ -145,11 +145,14 @@ data class State (
     ) {
         @Serializable
         data class PreferencesMetaData(
+            override var sampleRate: Float = 1f,
```

Comment:
@Nevazhnovu I'm not sure, but I believe whenever we add a new field to State, it has to be optional... otherwise, when deserializing data from local storage, it might throw an exception since the new field wasn't there...

Reply:
Regarding making sampleRate optional. I’ve double-checked, and the current solution seems working ok